tests/testthat/test-osmodel.R

```diff
@@ -0,0 +1,32 @@
+
+test_that("parametrize filled all the gaps", {
+    os_mod <- LogLogisticOs()
+    h_link <- HazardLink(
+        stan = StanModule(
+            functions = "bla",
+            data = "bla",
```

Comment:
Might be worth adding some more explicit tests to show that the merged stan objects come out as expected. I.e. after applying the character replacements  show that it equals a stanmodule built by hand with the replacements already applied.

Reply:
Like this one ? 
https://github.com/Genentech/jmpost/blob/0123fd37eb9c92d64ab2b941767822157607c02b/tests/testthat/test-osmodel.R#L36